### PR TITLE
Text update

### PR DESCRIPTION
**Pull request by avis71**
The changes includes all Hebrew changes that were done on the google docs texts by Tami and Alon.
The english version is broken a bit (waiting for Alon's translation), approved by Tami.

### CODE REVIEW

File: src/intl/en.json [hunk 1]
```diff
@@ -71,11 +71,25 @@
     "tabTitle": "When and how can I come or participate?",
     "title": "When and how can I come or participate?",
     "subtitle": "To set a volunteering day or a personal retreat, or to come for a visit, please contact Einat:",
+    "sub1":"Groups visits: ",
+    "sub2":"Private Visits: ",
+    "sub3":"Consultat: " ,
```
Comment (ranbena) [inline src/intl/en.json]: Typo: Consultant? Consultancy?

File: src/intl/en.json [hunk 2]
```diff
@@ -71,11 +71,25 @@
     "tabTitle": "When and how can I come or participate?",
     "title": "When and how can I come or participate?",
     "subtitle": "To set a volunteering day or a personal retreat, or to come for a visit, please contact Einat:",
+    "sub1":"Groups visits: ",
```
Comment (ranbena) [inline src/intl/en.json]: No "s".

```suggestion
    "sub1":"Group visits: ",
```

File: src/intl/en.json [hunk 3]
```diff
@@ -71,11 +71,25 @@
...
+    "sub2":"Private Visits: ",
+    "sub3":"Consultat: " ,
+    "sub4":"Learning Days: ",
+    "sub5":"Retreats: ",
+    "sub6":"Comminity Gatherings: ",
```
Comment (ranbena) [inline src/intl/en.json]: Typo:
```suggestion
    "sub6":"Community Gatherings: ",
```

File: src/intl/en.json [hunk 4]
```diff
@@ -71,11 +71,25 @@
...
+    "sub7":"Howtos: ",
+    "sub8":"Buy: ",
```
Comment (ranbena) [inline src/intl/en.json]: Maybe "Shop" is better here?

File: src/intl/he.json [hunk 5]
```diff
@@ -55,51 +55,75 @@
...
-      "עץבעיר הוא מרחב למידה והתרחשות לכל מי שמחוברת אליו, ויש דרכים שונות לחיבור. העבודה בגינה עם אלון והמפגשים הקהילתיים שלנו פתוחים לכול.",
+      "עץבעיר הוא מרחב למידה והתרחשות, הכולל את הדירה האקולוגית שבכיכר ביאליק וסביבה שלוש גינות בשטחי ציבור.",
+      "יש דרכים שונות להתחבר להתרחשות והלימוד בעץבעיר. מתקיימים אירועים קבועים, לצד אירועים עונתיים, ואפשר גם לבוא לימי התלמדות או מנוחה בתיאום אישי, או להזמין סיור אישי או קבוצתי.",
       "בימי שלישי ושישי מ-15:00 – אפשר להצטרף לעבודה בגינות עם אלון, רצוי בתיאום מראש.",
-      "פעם בשישה שבועות, בחילופי הזמנים, מתקיים מפגש קהילתי ביום שישי בבוקר, מ-11:00, בית פתוח להיכרות עם הבית ועם הקהילה שסביבו, וגם הזדמנות לרכוש מכל טוב העץ או להביא לו מתנות מהעודפים שלכן.",
-      "אנו מעדכנים על כל ההתרחשות בעלון שבועי ובקבוצת העדכונים בוואטסאפ.",
+      "פעם בחודש וחצי, בחילופי הזמנים, מתקיים מפגש קהילתי ביום שישי בבוקר, מ-11:00, זמן פתוח להיכרות עם הבית ועם הקהילה שסביבו, וגם הזדמנות לרכוש מכל טוב העץ או להביא לו מתנות מהעודפים שלכן.",
+      "אנו מעדכנים על כל ההתרחשות בעלוני דואֿ״ל, בקבוצת וואטסאפ כללית וכן בקבוצות וואטסאפ ייעודיות לכל סוג פעילות.",
+      "להצטרפות לקבוצת הוואטסאפ <whatsapp>דברים שקוריפ בעץ</whatsapp>",
```
Comment (ranbena) [inline src/intl/he.json]:
```suggestion
      "להצטרפות לקבוצת הוואטסאפ <whatsapp>דברים שקורים בעץ</whatsapp>",
```

File: src/intl/he.json [hunk 6]
```diff
@@ -55,51 +55,75 @@
...
     "subtitle": "לתיאום סדנה או סיור לקבוצה שלכן, דברו איתנו:",
-    "contact": "בטלפון: {phone} | בוואטסאפ: {whatsapp}"
+    "contact": "בטלפון: {phone} | בוואטסאפ: {whatsapp} "
   },
   "schedule": {
-    "tabTitle": "דברים שקורים בעץ",
-    "title": "דברים שקורים בעץ",
+    "tabTitle": "התרחשויות בעץ",
+    "title": "התרחשויות בעץ",
+    "subtitle": ":התרחשיות קבועות",
```
Comment (ranbena) [inline src/intl/he.json]: RTL stuff

```suggestion
    "subtitle": "התרחשיות קבועות:",
```

File: src/intl/en.json [hunk 7]
```diff
@@ -71,11 +71,25 @@
...
+      "To be translated soon...",
+      "To be translated soon...",
+      "To be translated soon...",
+      "You can also join us for a longer period of life and learning, be exposed to ecological practice, meet companions, rest and heal, whether in our seasonal retreats or by personal coordination.",
+      "To be translated soon...",
+      "To be translated soon...",
       "Our crazy little shop is open! You can set a time and come buy vinegar, bokashi, soap or other zero-waste products. Please visit the <shop>online shop</shop> that is gradually being enriched with more products.",
-      "You can also join us for a longer period of life and learning, be exposed to ecological practice, meet companions, rest and heal, whether in our seasonal retreats or by personal coordination."
+      "To be translated soon..."
```
Comment (ranbena) [inline src/intl/en.json:92]: The "schedule" section must also be updated - it's broken in En at the moment.

Comment (avis71) [reply src/intl/en.json:92]: ok, already notified Alon to be on that

**Pull request merged**